projects/lib/src/oauth-service.ts

```diff
@@ -1745,16 +1745,11 @@ export class OAuthService extends AuthConfig implements OnDestroy {
         location.origin +
         location.pathname +
         location.search
-          .replace(/code=[^&\$]*/, '')
-          .replace(/scope=[^&\$]*/, '')
-          .replace(/state=[^&\$]*/, '')
-          .replace(/session_state=[^&\$]*/, '')
-          .replace(/^\?&/, '?')
-          .replace(/&$/, '')
-          .replace(/^\?$/, '')
-          .replace(/&+/g, '&')
-          .replace(/\?&/, '?')
-          .replace(/\?$/, '') +
```

Comment:
The new way of matching will result in less situations that can break the format of the URL that's why these replace are not needed anymore.